test_settings.py.sample

```diff
@@ -1,2 +1,7 @@
-USERNAME = 'USERNAME'
-PRIVATE_KEY = 'PRIVATE_KEY'
+CONNECTION_OPTIONS = {
```

Comment:
We could - but I suspect it will be committed to github with real settings from time to time.  I think that was our initial intention to not change any github tracked file.